src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,68 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+// import { useState } from 'react';
+// import { Todo } from '../../types/Todo';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export enum FilteredOptions {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+interface Props {
+  query: string;
+  setQuery: (query: string) => void;
+  filteredField: FilteredOptions;
+  setFilterField: (filteredField: FilteredOptions) => void;
+}
+
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  setQuery,
+  filteredField,
+  setFilterField,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            value={filteredField}
+            data-cy="statusSelect"
+            onChange={event => {
+              setFilterField(event.target.value as FilteredOptions);
+            }}
+          >
+            <option value={FilteredOptions.All}>All</option>
+            <option value={FilteredOptions.Active}>Active</option>
+            <option value={FilteredOptions.Completed}>Completed</option>
```

Comment:
You can use the enum values with `Object.values` to create an array and then map through it to create options